### ISSUE TITLE
HTACCESS Recursion

### ISSUE THREAD

**Issue by matthewcrist** [Chicago/climatechangeisreal#2]
I'm trying to set the site up here @CityOfBoston. I'm getting this recursion error using the sample .htaccess. I haven't been able to successfully figure out a solution. Any thoughts?

> AH00124: Request exceeded the limit of 10 internal redirects due to probable configuration error. Use 'LimitInternalRecursion' to increase the limit if necessary. Use 'LogLevel debug' to get a backtrace., referer: http://138.197.13.68/

**Comment by henryjiang-sfgov** [comment]
San Francisco also experienced this issue when putting the rewrite rules in .htaccess.  We solved the issue by moving rewrite rules to the virtualhost block.

**Comment by chrisjcalderon** [comment]
That is correct. We added the rules at the end of the the httpd.conf.

@matthewcrist , I noticed that the site is running, however I'm seeing that one of the rules is not working properly for you (i.e. http://138.197.13.68/climate-change-science/)  this should be translated to
http://138.197.13.68/climate-change-science_.html and not /climate-change-science/_.html 

Let me know if the problem persists/need assistance.
